packages/components/src/components/tooltip/tooltip.lite.tsx

```diff
@@ -11,10 +17,35 @@ export default function DBTooltip(props: DBTooltipProps) {
 	const ref = useRef<HTMLDivElement>(null);
 	// jscpd:ignore-start
 	const state = useStore<DBTooltipState>({
+		initialized: false,
 		handleClick: (event: ClickEvent<HTMLElement>) => {
 			event.stopPropagation();
+		},
+		handleAutoPlacement: () => {
+			if (!ref) return;
+			handleDataOutside(ref);
```

Comment:
```suggestion
			if (ref) handleDataOutside(ref);
```

As you've extracted that method, we don't need that early return any more.